wp-includes/class-wp-service-workers.php

```diff
@@ -117,6 +117,31 @@ public function init() {
 			array( 'workbox-sw' )
 		);
 
+		$offline_page_id = (int) get_option( WP_Offline_Page::OPTION_NAME, 0 );
+		if ( $offline_page_id ) {
+
+			// Cache relevant theme assets.
+			$this->register_cached_route( '/wp-content/.*\.(?:css|gif|png|jpg|jpeg)', self::STRATEGY_STALE_WHILE_REVALIDATE );
```

Comment:
This will be problematic because it will store all uploaded files in the cache. We'll need to limit this to the actual theme directory URL.